nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -1476,13 +1517,23 @@ def build_transformer_config(self) -> TransformerConfig:
         recompute_method = self.cfg.get('activations_checkpoint_method', None)
         recompute_num_layers = self.cfg.get('activations_checkpoint_num_layers', None)
 
+        if not self.cfg.get('fp8', False):
```

Comment:
why do we want to have all these bools individually listed out?
lets make them like core where fp8 cfg variable is a string that is e4m3 or hybrid

Reply:
cool, can do that outside this PR